src/test/java/streamingapi/client/StreamingApiClientShould.java

```diff
@@ -78,15 +80,42 @@ public void throw_subscription_exception_when_creation_fails() throws IOExceptio
 		client.createSubscription(subscription, STREAMING_API_KEY);
 	}
 
+	@Test
+	public void get_success_status_when_delete_subscription() throws IOException {
+
+		Subscription subscription = getSubscription();
+
+		Subscription createdSubscription = client.createSubscription(subscription, STREAMING_API_KEY);
+
+		Response response = client.deleteSubscription(createdSubscription.getId(), STREAMING_API_KEY);
+
+		assertEquals(HTTP_NO_CONTENT_SUCCESS, response.getStatusCode());
+	}
+
+	@Test
+	public void get_not_found_status_when_deletion_fails() {
+
+		String invalidId = "AAA";
+
+		Response response = client.deleteSubscription(invalidId, STREAMING_API_KEY);
+
+		assertEquals(HTTP_NOT_FOUND, response.getStatusCode());
+	}
+
 	@Test
 	public void consume_events() throws Exception {
 
-		List<String> topics = asList(CLIENT_CREATED_TOPIC, DATA_ACCESS_STATE_CHANGED_TOPIC);
-		Subscription subscription = new Subscription(topics, TEST_APPLICATION_NAME);
+		Subscription subscription = getSubscription();
 
 		Subscription createdSubscription = client.createSubscription(subscription, STREAMING_API_KEY);
 
 		client.consumeEvents(createdSubscription.getId(), STREAMING_API_KEY);
 	}
 
+	private Subscription getSubscription() {
```

Comment:
Just a thought: I would rename this method to buildSubscription not to be confused with a GET http operation